### ISSUE TITLE
Fix Announcements for Browse Project Filters

### ISSUE THREAD

**Issue by michelled** [accessibility-in-action/platform#1423]
When filtering on the "browse all projects" page, filters are not announced properly for screenreaders. The first time a filter is applied it announces the number of filtered projects, if another filter is applied it doesn’t announce the number of filtered projects for the second time. Desired behaviour is to have the number of filtered projects announced every time a filter has been applied.
